_posts/2015-12-01-Client-First.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Client first
+excerpt: Think about your clients -- both humans and machines -- before starting to design your constraints
```

Comment:
"Think about your clients—both humans and machines—before starting to design your constraints"

Fixed em dashes.